### PR TITLE
#146 #239- fix boutton's issues, clean app.js to tour.js steps 0-5 ready

### PR DESCRIPTION
**Pull request by agamaloni**

### CODE REVIEW

File: static/js/app.js [hunk 1]
```diff
@@ -129,128 +129,106 @@ var SUBTYPE_STRING = [
...
-            app.map.setCenter(new google.maps.LatLng(lat, lon));
-        },
-        navigateEmpty: function() {
-            app.model.set("currentMarker", null);
-        }
-    });
+var checkStepsAdd = false;
+var tourLocation = 0;
+
+    $(function () {
```
Comment (danielhers) [inline static/js/app.js]: Why the indentation here?

File: static/js/tour.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ var tour = new Tour({
     <div class='popover-content'></div> \
     <nav class='popover-navigation-rtl'> \
         <div class='btn-group' role='group'> \
+            <button class='btn btn-default' data-role='prev'><< הקודם</button> \
```
Comment (danielhers) [inline static/js/tour.js]: Does not need to appear in the first one

File: static/js/tour.js [hunk 3]
```diff
@@ -20,54 +21,93 @@ var tour = new Tour({
     animation: true,
     title: "ברוכים הבאים!",
     placement: 'bottom',
-    content: '<p>ברוכים הבאים ל-ANYWAY. על גבי המפה ניתן לראות היכן התרחשו תאונות דרכים עם נפגעים.</p>',
-    onNext: function(tour){
+    content: '<p>ברוכים הבאים ל-ANYWAY, על גבי המפה ניתן לראות היכן התרחשו תאונות דרכים עם נפגעים.</p>',
```
Comment (danielhers) [inline static/js/tour.js]: A comma doesn't make sense to me here more than a dot, phrasing-wise

File: static/js/tour.js [hunk 4]
```diff
@@ -20,54 +21,93 @@ var tour = new Tour({
...
     }
-
   },
   {
     element:  '#pac-input',
     title: "חיפוש כתובת",
     animation: true,
     placement: 'bottom',
-    content: '<p>כאן תוכלו להזין את כתובתכם או כתובת שאתם נוהגים לבקר בה.</p></br>'+'<p> לחיצה על ENTER תיקח אתכם אל הכתובת. לדוגמא:</br>זבוטנסקי 74 פתח תקווה</p>',
+    content: '<p>כאן תוכלו להזין את כתובתכם או כתובת שאתם נוהגים לבקר בה.</p></br>'+'<p> לחיצה על ENTER תיקח אותכם אל הכתובת לדוגמא :</br>זבוטנסקי 74 פתח תקווה</p>',
```
Comment (danielhers) [inline static/js/tour.js]: Again, better phrasing is with the dot

File: static/js/tour.js [hunk 5]
```diff
@@ -20,54 +21,93 @@ var tour = new Tour({
...
-  steps: [
-  {
-    element:  $('#step2Component'),
-    placement: 'left',
+        {
+    element:  '#step3tour',
     title: "סינון לפי חומרה",
-    content: '<p>כאן ניתן להציג או להסתיר תאונות ברמות חומרה שונות. </br>לצורך הדוגמא נבטל את הצגתן של תאונות קלות עך ידי ביטול הסימון.</br> במידה ותבחרו להציג תאונות שמיקומן מוערך שימו לב שמידת ההערכה תופיע בשדה "עיגון" בפרטי התאונה ואייקון התאונה יהיה שקוף. </p>'
+    placement: 'bottom',
+    content: '<p>כאן ניתן להציג או להסתיר תאונות ברמות חומרה שונות. </br>לצורך הדוגמא נבטל את הצגתן של תאונות קלות עך ידי ביטול הסימון.</br> במידה ותבחרו להציג תאונות שמיקומן מוערך שימו לב שמידת ההערכה תופיע בשדה "עיגון" בפרטי התאונה ואייקון התאונה יהיה שקוף. </p>',
```
Comment (danielhers) [inline static/js/tour.js]: Typo: עך ידי
Please use a spell checker.

File: static/js/tour.js [hunk 6]
```diff
@@ -20,54 +21,93 @@ var tour = new Tour({
...
   },
 {
-    element:  $('#step2Component'),
+    element:  '#step4tour',
     title: "סינון לפי טווח תאריכים ",
-    content: '<p xmlns="http://www.w3.org/1999/html">כאן ניתן לבחור טווח תאריכים להצגת תאונות.</br>לנוחותכם ישנם קיצורי דרך לשנה ספציפית וכן אפשרות לבחור כל טווח תאריכים באופן ידני.</br> לצורך הדוגמא נבחר להציג תאונות מינואר 2006 עד ינואר 2014.</p>'
-  },
-]})
+    placement: 'bottom',
+    content: '<p>באמצעות התפריט ניתן לבחור טווח תאריכים ולהציג תאונות מרמות חומרה שונות</br>לנוחותכם ישנם קיצורי דרך לשנה ספציפית וכן אפשרות לבחור כל טווח תאריכים באופן ידני.</br> לצורך הדוגמא נבחר להציג תאונות מיאנור 2006 עד ינואר 2014.</p>'
```
Comment (danielhers) [inline static/js/tour.js]: Typo: יאנור

File: static/js/tour.js [hunk 7]
```diff
@@ -20,54 +21,93 @@ var tour = new Tour({
...
+function step3prev() {
+    tour.end();
+    tour.restart();
+    resetTour();
+    checkStepsAdd = false;
+    tour.goTo(1);
+}
+function tourAddInput() {
+    var a = document.getElementById('pac-input');
+    a.value = "זבוטנסקי 74, פתח תקווה";
```
Comment (danielhers) [inline static/js/tour.js]: זבוטינסקי should be ז'בוטינסקי

File: static/js/app.js (hunk 1 above)
Comment (agamaloni) [inline static/js/app.js]: you right.. should be at tour.js

File: static/js/tour.js (hunk 2 above)
Comment (agamaloni) [inline static/js/tour.js]: should be with change tour1 
tour defined below...  with new function ->
 function resetTour(){
    tour = new Tour(tour1);
    tour.addStep(tour1.getStep(0));
    tour.addStep(tour1.getStep(1));
    tour.init();
}
I faced some issues with using bootstrap tour as is over the google map , 
therefore the tour is set as "array" for new steps and i use tour1 as default to push new steps and reset the tour  
that gives me the ability to navigate the tour over a spot on map

File: static/js/app.js (hunk 1 above)
Comment (danielhers) [inline static/js/app.js]: So are you updating this pull request?

Comment (agamaloni) [inline static/js/app.js]: yes  :) thanks for the help.

2015-05-06 21:24 GMT+03:00 Daniel Hershcovich <notifications@github.com>:

Comment (agamaloni) [inline static/js/app.js]: later after the game...

2015-05-06 21:29 GMT+03:00 אגם אלוני <agamaloni@gmail.com>:

**Comment by agamaloni** [comment]
don't marge that comment !!!!
i had revert by mistake

**Comment by danielhers** [comment]
So you'll open another one?

**Comment by agamaloni** [comment]
yes i want to do it right this time ..
can you advice what i should do it?

**Comment by danielhers** [comment]
Start from the updated `master` branch, and then apply just the changes for the tour...

**Comment by danielhers** [comment]
I think you pushed some commits to the master by mistake. I reverted them.

**Pull request closed without merging**